source/assets/stylesheets/_core-print.scss

```diff
@@ -37,23 +37,18 @@ img { max-width: 100% !important; }
 select { background: white; }
 
 #global-header {
-  color: #000;
-  font-weight: bold;
-  font-size: 24px;
-  margin-bottom: 1em;
+  margin: 3px 0 1em; // 3px addtional space at top for countering -3px re-poositioning of #logo img
```

Comment:
‘addtional’ should be ‘additional’. ‘re-poositioning’ should be ‘re-positioning’